projects/plugins/crm/includes/ZeroBSCRM.DAL3.Helpers.php

```diff
@@ -5235,6 +5235,12 @@ function zeroBSCRM_get_invoice_settings(){
 	        'lang'                  => array(
 	                'invoice_number'    => zeroBSCRM_slashOut(__('ID', 'zero-bs-crm'),true),
 	                'invoice_date'      => zeroBSCRM_slashOut(__('Invoice date', 'zero-bs-crm'),true),
+				'invoice_status'          => zeroBSCRM_slashOut( __( 'Status', 'zero-bs-crm' ), true ),
```

Comment:
One option instead so that you don't have to hard-code the status would be to `array_merge()` from another location (e.g. `$zeroBSCRM_Conf_Setup`) after this code block. But any way it's done is messy since it's legacy code, and invoice statuses are currently not user-configurable anyway.